### ISSUE TITLE
OIDC: sso/complete view does use wrong redirection

### ISSUE THREAD

**Issue by bpedersen2** [ansible/awx#15439]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.
- [X] I am **NOT** reporting a (potential) security vulnerability. (These should be emailed to `security@ansible.com` instead.)

### Bug Summary

Current dev branch  @43a3d4a394f4fe6a4ba2489d9aca37cfba1c1221

generic oidc login shows a server error on redirect to <awxurl>/sso/complete

See below for backtrace.

git grep shows this is the only place a reverse match for `ui' is attempted.

### AWX version

Current dev branch  @43a3d4a394f4fe6a4ba2489d9aca37cfba1c1221

### Select the relevant components

- [X] UI
- [X] UI (tech preview)
- [ ] API
- [ ] Docs
- [ ] Collection
- [ ] CLI
- [ ] Other

### Installation method

kubernetes

### Modifications

no

### Steps to reproduce

1. Deploy the version  and configure  generic oidc
2. Try to login
3. See server error

### Expected results

Successful login

### Actual results

Server error:

```
Traceback (most recent call last):
Mon, Aug 12 2024 12:26:20 pm
  File "/var/lib/awx/venv/awx/lib64/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
Mon, Aug 12 2024 12:26:20 pm
    response = get_response(request)
Mon, Aug 12 2024 12:26:20 pm
               ^^^^^^^^^^^^^^^^^^^^^
Mon, Aug 12 2024 12:26:20 pm
  File "/var/lib/awx/venv/awx/lib64/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
Mon, Aug 12 2024 12:26:20 pm
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
Mon, Aug 12 2024 12:26:20 pm
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Mon, Aug 12 2024 12:26:20 pm
  File "/usr/lib64/python3.11/contextlib.py", line 81, in inner
Mon, Aug 12 2024 12:26:20 pm
    return func(*args, **kwds)
Mon, Aug 12 2024 12:26:20 pm
           ^^^^^^^^^^^^^^^^^^^
Mon, Aug 12 2024 12:26:20 pm
  File "/var/lib/awx/venv/awx/lib64/python3.11/site-packages/django/views/generic/base.py", line 104, in view
Mon, Aug 12 2024 12:26:20 pm
    return self.dispatch(request, *args, **kwargs)
Mon, Aug 12 2024 12:26:20 pm
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Mon, Aug 12 2024 12:26:20 pm
  File "/var/lib/awx/venv/awx/lib64/python3.11/site-packages/awx/sso/views.py", line 38, in dispatch
Mon, Aug 12 2024 12:26:20 pm
    response = super(CompleteView, self).dispatch(request, *args, **kwargs)
Mon, Aug 12 2024 12:26:20 pm
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Mon, Aug 12 2024 12:26:20 pm
  File "/var/lib/awx/venv/awx/lib64/python3.11/site-packages/django/views/generic/base.py", line 143, in dispatch
Mon, Aug 12 2024 12:26:20 pm
    return handler(request, *args, **kwargs)
Mon, Aug 12 2024 12:26:20 pm
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Mon, Aug 12 2024 12:26:20 pm
  File "/var/lib/awx/venv/awx/lib64/python3.11/site-packages/django/views/generic/base.py", line 257, in get
Mon, Aug 12 2024 12:26:20 pm
    url = self.get_redirect_url(*args, **kwargs)
Mon, Aug 12 2024 12:26:20 pm
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Mon, Aug 12 2024 12:26:20 pm
  File "/var/lib/awx/venv/awx/lib64/python3.11/site-packages/awx/sso/views.py", line 25, in get_redirect_url
Mon, Aug 12 2024 12:26:20 pm
    url = reverse('ui:index')
Mon, Aug 12 2024 12:26:20 pm
          ^^^^^^^^^^^^^^^^^^^
Mon, Aug 12 2024 12:26:20 pm
  File "/var/lib/awx/venv/awx/lib64/python3.11/site-packages/django/urls/base.py", line 82, in reverse
Mon, Aug 12 2024 12:26:20 pm
    raise NoReverseMatch("%s is not a registered namespace" % key)
Mon, Aug 12 2024 12:26:20 pm
django.urls.exceptions.NoReverseMatch: 'ui' is not a registered namespace
```

Manually navigation to the index page show login succeeded.

**Comment by bpedersen2** [comment]
The url namespace got removed in a41766090e30d117e39d4646f67f20a8c2c00841

**Comment by bpedersen2** [comment]
Fixed by #15456
